README.md

```diff
@@ -29,12 +29,8 @@ pip install -r requirements_dev.txt
 createdb iati-website
 cp settings/local.py.example settings/local.py
 
```

Comment:
Bit confused about how these changes have made it into this branch. Perhaps `add-about-models` has not been updated with `add-page-models`??